AGGREGATE.md

```diff
@@ -315,6 +308,25 @@ future versions of the API), the endpoint URL should also change.
 **Note:** The browser may need some mechanism to ensure that the same set of
 keys are delivered to different users.
 
+#### Optional: extended debugging reports
+
+If [debugging](https://github.com/WICG/conversion-measurement-api/blob/main/EVENT.md#optional-extended-debugging-reports)
+is enabled, additional debug fields will be present in aggregatable reports.
+The `source_debug_key` and `trigger_debug_key` fields match those in the
+event-level reports. The `debug_cleartext_payload` field contains the
```

Comment:
Shall we mention that `debug_cleartext_payload` and `debug_mode` will only be present if both source and trigger debug keys are present?

Reply:
Thanks, updated!